robot/web/web.go

```diff
@@ -818,6 +844,17 @@ func (svc *webService) initAuthHandlers(listenerTCPAddr *net.TCPAddr, options we
 					handler.Type,
 					rpc.MakeSimpleMultiAuthHandler(authEntities, locationSecrets),
 				))
+			case oauth.CredentialsTypeOAuthWeb:
+				webOauthOptions := oauth.WebOAuthOptions{
+					AllowedAudiences: handler.WebOauthConfig.AllowedAudiences,
+					KeyProvider:      jwks.NewStaticJWKKeyProvider(handler.WebOauthConfig.ValidatedKeySet),
+					EntityVerifier: func(ctx context.Context, entity string) (interface{}, error) {
+						return map[string]interface{}{"email": entity}, nil
```

Comment:
maybe a comment here as to why email is entity

Reply:
Added. I wanted email to be consistent with app. Once we start doing interesting things on the RDK for authorization etc.. we'll probably add a similar state structure like we have in the app.

I have a companion PR working now: https://github.com/viamrobotics/app/pull/950 the app will auto add to the AuthHandlers. Im open to debate if the RDK should be doing that or just accepting the auth handler as is. To be having the cloud edit the config makes it much more visible and easier to reason about then trying to understand the underlying logic in the RDK if managed or not.

Reply:
I agree. I regret my past decisions :)

Reply:
Ack :)